grid/chat/channels/profile.supr

```diff
@@ -0,0 +1,191 @@
+name = "chat/channels"
+version = "1.0.0"
+
+"""
+Get Channels
+Retrieve paginated list of channels, e.g. text channels in slack or discord
+"""
+usecase GetChannels {
+  input {
+    """
+    Server
+    Representation of a server. Can be guild or server, depending on the provider capability.
```

Comment:
Did we establish using `server` as the identifier here? I think in most Slack-like applications it's called "workspace" (I know about Slack, Rocket.Chat, Mattermost). Calling it "server" can be also confusing for apps supporting self-hosted deployments _and_ workspaces. So I'd consider renaming this to `workspace` or perhaps more abstract `space`. But maybe @janhalama has a different opinion here?

Reply:
I agree that server is not the best option. I would go with `workspace`.

Reply:
Alright, I will also open new PR with `chat/workspaces` profile for getting workspaces (and guilds in case of discord).